frontend/src/pages/modelServing/screens/projects/InferenceServiceModal/DataConnectionExistingField.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import {
+  FormGroup,
+  Select,
+  SelectOption,
+  Stack,
+  StackItem,
+  TextInput,
+} from '@patternfly/react-core';
+import { DataConnection, UpdateObjectAtPropAndValue } from 'pages/projects/types';
+import { CreatingInferenceServiceObject } from '../../types';
+import { getDataConnectionDisplayName } from 'pages/projects/screens/detail/data-connections/utils';
+import '../../../../projects/screens/detail/storage/ManageStorageModal.scss';
```

Comment:
We should never import styles like this. This is probably fine for now, but don't go hunting for other resources styles. If you need the styles, you dupe it or we setup some custom state of styles.